eva_submission/eload_ingestion.py

```diff
@@ -160,6 +160,11 @@ def load_from_ena(self):
             self.eload_cfg.set(self.config_section, 'ena_load', value='failure')
             raise e
 
+    def _copy_file(self, source_path, target_dir):
+        target_path = target_dir.joinpath(source_path.name)
+        if not target_path.exists():
+            shutil.copyfile(source_path, target_path)
```

Comment:
Can we warn about the files existing here? 
We should also check if the best strategy is to overwrite by default.

Reply:
Agree about warning,  I think I didn't make it overwrite by default because I imagined we might occasionally need to make manual fixes and rerun ingestion, but I'm now getting the feeling that we would never ever want to do that...